stubs/pygit2/pygit2/_pygit2.pyi

```diff
@@ -0,0 +1,773 @@
+from collections.abc import Iterator
+from io import IOBase
+from typing import Any, Literal, final, overload
+from typing_extensions import TypeAlias
+
+from . import Index
+from .enums import (
+    ApplyLocation,
+    BranchType,
+    DiffFind,
+    DiffFlag,
+    DiffOption,
+    DiffStatsFormat,
+    FileMode,
+    FileStatus,
+    MergeAnalysis,
+    MergePreference,
+    ObjectType,
+    Option,
+    ReferenceFilter,
+    ReferenceType,
+    ResetMode,
+    SortMode,
+)
+from .filter import Filter
+
+GIT_APPLY_LOCATION_BOTH: int
```

Comment:
Optionally, you can set the actual values here. (To be clear, this isn't required for merging, especially as there is a ginormous number of constants in this library.)

```suggestion
GIT_APPLY_LOCATION_BOTH: Final = 2
```